clients/vault/src/oracle/collector/envelopes_handler.rs

```diff
@@ -43,9 +51,6 @@ impl ScpMessageCollector {
 				txset_hash_map.insert(txset_hash, slot);
 				// let's request the txset from Stellar Node
 				overlay_conn.send(StellarMessage::GetTxSet(txset_hash)).await?;
-
-				// check if we need to transfer the map to a file
-				self.check_write_envelopes_to_file(slot)?;
```

Comment:
Is there a special reason why you remove this? It's not bad since we talked about removing the 'storing to file logic' overall anyways, but maybe this is not the right pr to do it.

Reply:
sure. you are right.

Reply:
Done.